torch/csrc/lazy/python/init.cpp

```diff
@@ -307,6 +307,15 @@ void initLazyBindings(PyObject* module) {
 #endif // !(defined(FBCODE_CAFFE2) || defined(OVRSOURCE))
         return result;
       });
+  lazy_ts_backend.def("_get_latest_computation_graph", []() {
```

Comment:
should this be removed?

Reply:
I added it as a tool to verify that the generator was included as part of the TorchScript graph in the LTC flow. I think its a useful utility to have going forward. Thoughts @wconstab?